packages/gatsby/src/utils/source-nodes.js

```diff
@@ -31,11 +31,12 @@ function discoverPluginsWithoutNodes(storeState) {
   return _.difference(nodeCreationPlugins, nodeOwners)
 }
 
-module.exports = async ({ parentSpan } = {}) => {
+module.exports = async ({ webhookBody = {}, parentSpan } = {}) => {
```

Comment:
I have question about setting empty object as default - I wonder if it would be better to not do that - as user have to do more checks than just looking if `webhookBody` exists. Also body could technically not be plain object, but an array (or whatever) - so assumption that it will be object might be more problematic in those cases as well?

Reply:
For sure. I was thinking of the case where e.g. `if (webhookBody.updates)` or something, and didn’t want to _have_ to guard against null. In other words, I think it’s likely this would be a nested structure passed from the CMS, so they’d check for a nested property instead of the existence of the object itself.

I’m happy to do either though.

Reply:
I would not set default, but this is not the hill I will die on ;) I leave that decision to You, just wanted to showcase edge cases.